src/foam/dao/NodeFileJournal.js

```diff
@@ -0,0 +1,100 @@
+foam.CLASS({
```

Comment:
Please insert:
/**
 * @license
 * Copyright 2018 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */